tickets/admin.py

```diff
@@ -97,6 +107,73 @@ def convert_to_assistant(self, request, queryset):
 
     convert_to_assistant.short_description = "Convert to/delete assistant"
 
+    # if for some reason you need to change ects of someone
+    def set_max_ects(self, request, queryset):
+        for obj in queryset:
+            from editions.models import Track
+            track = Track.objects.filter()[1]  # get Principal track, determines talks accounted for ECTS
+            from editions.models import Session
+            from TryIT.settings_global import EDITION_YEAR
+            number_of_sessions = Session.objects \
+                .filter(edition__year=EDITION_YEAR) \
+                .filter(track=track).count()
+
+            maximum_ects = 3.0 if obj.active else 2.0
+
+            ects_by_session = 2.0 / number_of_sessions
+
+            needed_talks = abs((maximum_ects - obj.ects) % ects_by_session)
```

Comment:
method abs is unnecessary. 
Modulo operation always return positive number.

Its not clear what obj.ects is.